src/main/scala/it/pagopa/interop/attributeregistryprocess/common/readmodel/ReadModelQueries.scala

```diff
@@ -0,0 +1,101 @@
+package it.pagopa.interop.attributeregistryprocess.common.readmodel
```

Comment:
This file is correlated to PIN-2958 and its logic belongs to the process you used as a seed.